package.json

```diff
@@ -11,11 +11,13 @@
   },
   "resolutions": {
     "eslint": "^6.6.0",
-    "@polkadot/api": "8.1.1",
-    "@polkadot/util": "9.0.1",
-    "@polkadot/keyring": "9.0.1",
-    "@polkadot/util-crypto": "9.0.1",
-    "@polkadot/wasm-crypto": "6.1.1",
+    "@polkadot/api": "^8.7.1",
```

Comment:
Let's hold off on updating these `@polkadot/` packages. They should be synced with the `@webb-tools/` packages.